content/applications/finance/expenses/expense_reports.rst

```diff
@@ -0,0 +1,109 @@
+===============
+Expense reports
+===============
+
+When expenses are ready to submit (such as at the end of a business trip, or once a month), an
+*expense report* needs to be created. Navigate to the main :menuselection:`Expenses` app dashboard,
+which displays a default :guilabel:`My Expenses` view, or go to :menuselection:`Expenses app --> My
+Expenses --> My Expenses`.
+
+Expenses are color coded by status. Any expense with a status of :guilabel:`To Report` (expenses
+that still need to be added to an expense report) the text appears in blue. All other statuses
+(:guilabel:`To Submit`, :guilabel:`Submitted`, and :guilabel:`Approved`) the text appears in black.
+
+Create an expense report
+========================
+
+First, select each individual expense to be added to the report on the :guilabel:`My Expenses`
+dashboard, by ticking the checkbox next to each entry, or quickly select all the expenses in the
+list by ticking the checkbox next to :guilabel:`Expense Date`.
+
+Another way to quickly add all expenses that are not on an expense report is to click
+:guilabel:`Create Report` without selecting any expenses, and Odoo automatically selects all
+expenses with a status of :guilabel:`To Submit` that are not already on a report.
+
+.. image:: expense_reports/create-report.png
+   :align: center
+   :alt: Select the expenses to submit, then create the report.
+
+.. note::
+   Any expense can be selected from the :guilabel:`My Expenses` list, regardless of status. The
+   :guilabel:`Create Report` button is visible as long as there is a minimum of one expense with a
+   status of :guilabel:`To Report` selected. When the :guilabel:`Create Report` button is clicked,
+   only expenses with a status of :guilabel:`To Submit` that are *not* currently on another expense
+   report will appear in the newly created expense report.
+
+Once the expenses have been selected, click the :guilabel:`Create Report` button. The new report
+appears with all the expenses listed in the :guilabel:`Expense` tab. If there is a receipt attached
+to an individual expense, a :icon:`fa-paperclip` :guilabel:`(paperclip)` icon appears next to the
+:guilabel:`Customer to Reinvoice` and :guilabel:`Analytic Distribution` columns.
+
+When the report is created, the date range for the expenses appears in the :guilabel:`Expense Report
+Summary` field by default. It is recommended to edit this field with a short summary for each report
+to help keep expenses organized. Enter a description for the expense report, such as `Client Trip
+NYC`, or `Office Supplies for Presentation`, in the :guilabel:`Expense Report Summary` field.
+
+Next, select a :guilabel:`Manager` from the drop-down menu to assign a manager to review the report.
+If needed, update the :guilabel:`Journal` field, using the drop-down menu.
```

Comment:
Do you think it might be beneficial to mention the 'Employee', 'Paid By', and 'Company' fields? Just a quick explanation of them may be helpful to readers.